src/ECER.Clients.RegistryPortal/ECER.Clients.RegistryPortal.Server/PortalInvitations/PortalInvitationsEndpoints.cs

```diff
@@ -0,0 +1,24 @@
+﻿using ECER.Managers.Admin.Contract.PortalInvitations;
+using ECER.Utilities.Hosting;
+using MediatR;
+using Microsoft.AspNetCore.Http.HttpResults;
+
+namespace ECER.Clients.RegistryPortal.Server.References;
+
+public class PortalInvitationsEndpoints : IRegisterEndpoints
+{
+  public void Register(IEndpointRouteBuilder endpointRouteBuilder)
+  {
+    endpointRouteBuilder.MapGet("/api/PortalInvitations/{token?}", async Task<Results<Ok<ReferenceQueryResult>, BadRequest<string>>> (string? token, IMediator messageBus, HttpContext httpContext, CancellationToken ct) =>
+    {
+      if (token == null) return TypedResults.BadRequest("Token is required");
+      var portalInvitationQueryResults = await messageBus.Send(new InviteLinkQuery(token), ct);
+      var portalInvitation = portalInvitationQueryResults.Items.SingleOrDefault();
+      var registrant = portalInvitation!.Registrant!.Profile;
```

Comment:
TMI - you want to return from the handler only the information you need - you're implementing a use case for this case, projection should happen at the manager handler level